api/src/main/scala/io/codegalaxy/api/chapter/ChapterApi.scala

```diff
@@ -4,5 +4,5 @@ import scala.concurrent.Future
 
 trait ChapterApi {
 
-  def getChapters(topic: String): Future[List[ChapterRespData]]
```

Comment:
We can remove this method from the implementation (`CodeGalaxyApiClient`) as well then.

Reply:
Let's keep it. It works but way more lightweight, so it can be used later